README.md

```diff
@@ -36,11 +36,11 @@ apply(plugin = "build.buf")
 ```
 
 When applied the plugin creates tasks:
```

Comment:
Can we add a link above to instructions for the line

> By default this plugin assumes that Buf is configured for the project root (with or without a workspace `buf.work.yaml`)

Reply:
Hm, you're suggesting that we have more detailed set up instructions for when the user does not have buf yamls?

Reply:
I'm suggesting linking out to the instructions for setting up buf yaml files in case the reader is not aware of what they are or how to configure them. Don't think you need to duplicate information from those docs

Reply:
Gotcha

Reply:
Added a link to buf docs getting started in the description doc.